pylanetary/utils/core.py

```diff
@@ -204,4 +205,42 @@ def convolve_with_beam(data, beam):
     else:
         psf = beam
     return convolution.convolve_fft(data, psf)
-    
\ No newline at end of file
+
+
+class Planet:
+    """
+    Class will initiate planet object with attributes from yaml file of matching
+    string name.
+    """
+    def __init__(self, name):
+        """
+        Input string of planet name to initiaite object and find yaml file of same
+        matching string. Optional input of paameters from
+
+        Parameters
+        ----------
+        name: str
+            Planet name string, example "Jupiter' will load Jupiter.yaml
+        """
+        self.name = name
+        with importlib.resources.open_binary(planet_info, f"{self.name}.yaml") as file:
+            yaml_bytes = file.read()
+            data = yaml.safe_load(yaml_bytes)
+
+            # basic information and rewrite name
+            self.name = data['planet']['name']
+            self.mass = data['planet']['mass'] * u.kg
+            self.Re = data['planet']['Re'] * u.km
```

Comment:
for consistency with the rest of the code, please use self.req and self.rpol.  This should also be changed in the yaml files themselves